File: backend/src/main/java/com/messmate/backend/dto/response/AdminMealDashboardResponse.java
package com.messmate.backend.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalDate;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class AdminMealDashboardResponse {
    private LocalDate targetDate;
    private int totalActiveMembers;

    private int todayLunchYes;
    private int todayLunchNo;
    private int todayDinnerYes;
    private int todayDinnerNo;
    private int totalLunchMeals;
    private int totalDinnerMeals;
    private int totalMealUnits;

    private String lunchVotingStatus; // "OPEN" or "CLOSED"
    private String dinnerVotingStatus; // "OPEN" or "CLOSED"

    private java.util.List<MemberMealDetailResponse> memberDetails;
}

File: android/app/src/main/java/com/messmate/android/data/meal/AdminMealDashboardResponse.kt
package com.messmate.android.data.meal

data class AdminMealDashboardResponse(
    val todayLunchYes: Int,
    val todayLunchNo: Int,
    val todayDinnerYes: Int,
    val todayDinnerNo: Int,
    val totalLunchMeals: Int,
    val totalDinnerMeals: Int,
    val totalMealUnits: Int,
    val lunchVotingStatus: String,
    val dinnerVotingStatus: String
)
